CoreGraphUtilities/src/au/gov/asd/tac/constellation/graph/utilities/planes/DragDropList.java

```diff
@@ -205,7 +204,7 @@ public int getListIndex(final int elementIndex) {
         public MyDragListener(final DragDropList list) {
             this.list = list;
             ds = new DragSource();
-            DragGestureRecognizer dgr = ds.createDefaultDragGestureRecognizer(list, DnDConstants.ACTION_MOVE, this);
+            ds.createDefaultDragGestureRecognizer(list, DnDConstants.ACTION_MOVE, this);
```

Comment:
Still required to create the dragGestureRecognizer which is used later but the variable itself can be removed because its never read from.